client/device/duoadminpanel/delete.go

```diff
@@ -0,0 +1,20 @@
+package duoadminpanel
+
+import (
+	"context"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+)
+
+type DeleteInput struct {
+	Uid string
+}
+
+type DeleteOutput = device.DeleteOutput
+
+func Delete(ctx context.Context, client http.Client, deleteInp DeleteInput) (*DeleteOutput, error) {
+
+	client.Logger.Println("deleting duo admin panel")
```

Comment:
I'll raise a story to add logging levels throughout the codebase :)

Reply:
👍

Reply:
See: LH-75365